cfme/tests/cli/test_appliance_console.py

```diff
@@ -1423,8 +1422,15 @@ def test_appliance_console_apache_reload_log_rotate():
         initialEstimate: 1/12h
         startsin: 5.10
         testtype: structural
+        testSteps:
+            1. execute command 'cat  /etc/logrotate.d/miq_logs.conf'
+        expectedResults:
+            1. "bin/apache reload"  content should not be in output
     """
-    pass
+    command = "cat  /etc/logrotate.d/miq_logs.conf"
+    result = appliance.ssh_client.run_command(command)
+    assert result.success
+    assert "bin/apache reload" not in result.output, "There is some issue in apache reload"
 
```

Comment:
**Optional:**
Do we have exact reason for why apache reload gets failed? So that, we can add proper reason for assertion fail.

Reply:
Removed assert ambiguous message